docs/configuration/session.md

```diff
@@ -32,6 +32,12 @@ session:
   # The inactivity time in seconds before the session is reset.
   inactivity: 300 # 5 minutes
 
+  # The remember me duration.
+  # Value of 0 disables remember me.
+  # Value is in seconds, or duration notation. See: https://docs.authelia.com/configuration/session.html#duration-notation
+  # Longer periods are less secure. Currently the default is 1M or 1 month.
```

Comment:
```suggestion
  # Longer periods are considered less secure because a stolen cookie will last longer giving attackers more time to spy or attack. Currently the default is 1M or 1 month.